Settings tab doesn't replace temporary tab

monsterkodi opened the pull request:
In my optinion, the Atom Settings page should not replace the current temporary tab

ddavison commented:
hey monster, thanks for the patch!  would you write a test for this, though? :smiley:

monsterkodi commented:
hi ddavison,
I made another pull request where I modified the settings tab detection (using the uri instead of tab name) and changed the test to check for the new behaviour. I don't know much about atom development, so I hope it is what you expected.

ddavison commented:
perfect! thanks so much :smile:
